code/modules/clothing/clothing.dm

```diff
@@ -625,7 +625,8 @@ BLIND     // can't see anything
 		/obj/item/device/radio,
 		/obj/item/clothing/mask,
 		/obj/item/storage/backpack/duffelbag/guncase,
-		/obj/item/implant/carrion_spider/holographic)
+		/obj/item/implant/carrion_spider/holographic,
+		/obj/item/melee/energy/spear)
```

Comment:
This is not needed as it already has obj.item.melee

Reply:
Thank you catto, I must've been blind